demo/src/test/java/com/example/test/pages/KBBMOOCProject/WhatsMyCarWorthPage/WhatsMyCarWorthPage.java

```diff
@@ -0,0 +1,136 @@
+package com.example.test.pages.KBBMOOCProject.WhatsMyCarWorthPage;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import com.example.test.pages.KBBMOOCProject.KbbBasePage;
+
+
+public class WhatsMyCarWorthPage extends KbbBasePage {
+    
+    public WhatsMyCarWorthPage(WebDriver driver) {
+        super(driver);
+    }
+    
+
+    WebElement txtVIN = keyword.findWebElementByXpath("//div[@id='vinNumberInput']//input");
+    WebElement txtEmail = keyword.findWebElementByXpath("//div[@id='vinEmailInput']//input");
+    WebElement btnGo = keyword.findWebElementByXpath("//button[@data-testid='vinSubmitBtn']");
+    WebElement chkLicensePlate = keyword.findWebElementByXpath("//div[text()='License Plate']");
+    WebElement chkMakeModel = keyword.findWebElementByXpath("//div[text()='Make/Model']");
+    //#endregion
+
+    //#region page actions
+    /**
+     * Option VIN
+     * @throws InterruptedException
+     */
+    public void sendKeysToVINTextbox(String vin) throws InterruptedException {
+        keyword.sendKeys(txtVIN, vin);
+    }
+    public void sendInvalidKeysToVINTextbox(String errorMessage) throws InterruptedException {
+        keyword.sendKeys(txtVIN, errorMessage);
+    }
+    public void sendKeysToEmailTextbox(String email) throws InterruptedException {
+        keyword.sendKeys(txtEmail, email);
+    }
+    public void clickOnVINGoButton() throws InterruptedException {
+        keyword.scrollAndWaitToClick(btnGo);
+    }
+    public String getErrorMessageString(){
+        WebElement spanErrorMessage = keyword.findWebElementByXpath("//span[@data-automation='error-message']");
+        String errorMessageString = spanErrorMessage.getText();
+        
+        return errorMessageString;
+    }
+    public boolean compareErrorMessage(String errorMessage) {
+        if(getErrorMessageString().equals(errorMessage) ){
+            return true;
+        }
+        return false;
+    }
+
+    /**
+     * Option License Plate
+     * @throws InterruptedException
+     */
+    public void clickOnLicensePlateCheckbox() throws InterruptedException {
+        keyword.scrollAndWaitToClick(chkLicensePlate);
+    }
+    public void sendKeysToTextboxes(String plate, String invalidEmail) throws InterruptedException {
+        WebElement txtPlate = keyword.findWebElementByXpath("//input[@data-lean-auto='plateInput']");
+        WebElement txtEmail = keyword.findWebElementByXpath("//input[@data-lean-auto='emailLpInput']");
+        keyword.sendKeys(txtPlate, plate);
+        keyword.sendKeys(txtEmail, invalidEmail);
+    }
+    public void selectState() throws InterruptedException {
+        WebElement ddlState = keyword.findWebElementByXpath("//select[@placeholder='State']");
+        keyword.scrollAndWaitToClick(ddlState);
+        
+    }
+    public void selectHIoption() throws InterruptedException{
+        WebElement optHI = keyword.findWebElementByXpath("//option[@value='HI']");
```

Comment:
should write the web element locator in separate line then call into a method/function